flatpak_builder_lint/staticfiles/exceptions.json

```
@@ -1004,6 +1004,7 @@
         "finish-args-flatpak-spawn-access": "the app predates this linter rule"
     },
     "com.syntevo.SmartSynchronize": {
+        "finish-args-flatpak-spawn-access": "Required access to work properly",
```

```suggestion
        "finish-args-flatpak-spawn-access": "the app predates this linter rule",
```

The current description doesn't really explain anything. I'd rather have it generic "it's prior work" than this vague.